integration/js/script/run-test

```diff
@@ -77,67 +77,116 @@ cd ../
 . ~/.bashrc
 
 case $1 in
-   audio)
-      r audio_test.config.json
-      r app_quit_audio_test.config.json
-      r meeting_leave_audio_test.config.json
-      ;;
-   video)
-      r video_test.config.json
-      r video_test_processor.config.json
-      #  TODO: Re-enable simulcast test after it has been fixed
-      # r video_test_simulcast.config.json
-      r app_quit_video_test.config.json
-      r meeting_leave_video_test.config.json
-      r reconnection_test.config.json
-      ;;
-   video_test_app)
-      r test_app_video_test.config.json
-      r test_app_video_pin_test.config.json
-      ;;
-   meeting_end)
-      r meeting_end_test.config.json
-      ;;
-   content_share_test_suite_one)
-      r content_share_screen_capture_test.config.json
-      r meeting_leave_content_share_test.config.json
-      r content_share_video_test.config.json
-      ;;
-   content_share_test_suite_two)
-      r content_share_only_allow_two_test.config.json
-      r content_share_join_later_test.config.json
-      r app_quit_content_share_test.config.json
-      ;;
-   data_message)
-      r data_message_test.config.json
-      ;;
-   meeting_readiness_checker)
-      r meeting_readiness_checker_test.config.json
-      ;;
-   messaging)
-      r messaging_session_test.config.json
-      ;;
-   transcription)
-      r transcription_test.config.json
-      ;;
-   background_blur)
-      r background_blur_test.config.json
-      ;;
-   echo_reduction)
-      r echo_reduction_test.config.json
+   integration)
+   case $2 in
```

Comment:
I think you need to correct the syntax on this case here, seeing this in GitHub with this file:
![Screen Shot 2021-11-23 at 3 17 46 PM](https://user-images.githubusercontent.com/61809205/143144017-4972a8cd-2b33-4577-bba3-f795f31cf666.png)

Reply:
This is syntax no issues, resolving this.